typescript/packages/well-log-viewer/src/components/WellLogView.tsx

```diff
@@ -306,7 +314,7 @@ function addPinnedValueOverlay(instance: LogViewer, parent: WellLogView) {
 }
 
 export interface WellPickProps {
-    wellpick: WellLog; // JSON Log Format
+    wellpick: WellLogSet; // JSON Log Format
```

Comment:
any better name ?

Reply:
Probably, I just replaced WellLog here since that type is now deprecated, anything else is arguably out-of-scope for this PR.  

ON that note however, I don't see why the well-pick property needs to receive a full well-log structure? All it does is draw a line per "curve", right? Wouldnt an array of `{ name, md, color }` objects be more than sufficient and less cumbersome?

Reply:
It uses more in getWellPicks, no ?

Reply:
Probably correct, I didn't look into that part of the code that much... but regardless, that doesn't really matter for this PR so I dont think we should discuss it further.

Regarding the initial comment, I think this line can be left as is: I just migrated the now deprecated `WellLog` type, nothing has mechanically changed, and changes to the well-pick logic would be out-of-scope, in my opinion.

Reply:
ok